src/common/crypt.ts

```diff
@@ -40,13 +40,34 @@ const preCrypt = async (path): Promise<void> => {
   }
 }
 
-const getAllSecretFiles = async (path) => {
+export const getAllSecretFiles = (relativePaths: Array<string>, encryption: boolean, suffix: string) => {
+  /** Use cases
+   * 1. encryption for the first time when .dec does not exists
+   * 2. encryption after bootstraping files, so they also do not have .dec
+   * 3. encryption of files with .dec
+   * 4. decryption of secrets.*.yaml files
+   */
+
   const d = terminal(`common:crypt:getAllSecretFiles`)
-  const files = (await readdirRecurse(`${path}/env`, { skipHidden: true }))
-    .filter((file) => file.endsWith('.yaml') && file.includes('/secrets.'))
-    .map((file) => file.replace(`${path}/`, ''))
-  d.debug('getAllSecretFiles: ', files)
-  return files
+  let filteredFiles: Array<string> = []
```

Comment:
I don't see the point of this as sops already works this way. Before it was fine, so why try to suddenly emulate sops behaviour? Am I missing something?